.github/workflows/unversioned_pages.yml

```diff
@@ -0,0 +1,36 @@
+name: Copy unversioned pages
+
+on:
+  push:
+    branches:
+      - gh-pages
+    paths:
+      - 'dev/developers/**'
+      - 'dev/index.html'
+  workflow_dispatch:
+
+jobs:
+  copy-pages:
+    name: Copy unversioned pages
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone artifacts repo
+        uses: actions/checkout@v4
+
+      - name: Copy pages
+        run: |
+          ls -la
+          cp -Rv ./dev/developers/ ./stable/
+          cp -v ./dev/index.html ./stable/
```

Comment:
Is this really a good idea? Once we updated theme, the copied pages will be with new theme, when rest will be with old one. 
I think that we should use same approach as in banner. So we should have JS that will download page from dev, extract content from dev and then replace current with one from dev.

Reply:
Do you mean: a javascript that, every time the developer docs are accessed by a reader, replaces the contents on the fly? I have not explored this possibility, I can take a look 🤔